Add details about datasets with restricted access (#126)
mila-iqia/mila-docs: docs/Information_storage.rst

``datasets`` contains curated datasets to the benefit of the Mila community.
To request the addition of a dataset or a preprocessed dataset you think could
benefit the research of others, you can fill `this form
<https://forms.gle/vDVwD2rZBmYHENgZA>`_. Datasets in ``datasets/restricted``

> are restricted and require an explicite request to gain access. Please `submit
> a support ticket <https://milaquebec.freshdesk.com/a/tickets/new>`_ mentionning

Reviewer: ```suggestion
are restricted and require an explicit request to gain access. Please `submit
a support ticket <https://milaquebec.freshdesk.com/a/tickets/new>`_ mentioning
```

> <https://forms.gle/vDVwD2rZBmYHENgZA>`_. Datasets in ``datasets/restricted``
> are restricted and require an explicit request to gain access. Please `submit
> a support ticket <https://milaquebec.freshdesk.com/a/tickets/new>`_ mentioning
> the dataset's access group (ex.: ``scannet_users``) and your cluster's
> username. You can find the dataset's access group by listing the content of
> ``/network/datasets/restricted`` with the `ls command
> <https://cli-cheatsheet.readthedocs.io/en/latest/#ls>`_.

Author: ```suggestion
<https://forms.gle/vDVwD2rZBmYHENgZA>`_.

Datasets in ``datasets/restricted`` are restricted and require an explicit
request to gain access. Please `submit a support ticket
<https://milaquebec.freshdesk.com/a/tickets/new>`_ mentioning the dataset's
access group (ex.: ``scannet_users``), your cluster's username and the
approbation of the group owner. You can find the dataset's access group by
listing the content of ``/network/datasets/restricted`` with the `ls command
<https://cli-cheatsheet.readthedocs.io/en/latest/#ls>`_.
```

> <https://forms.gle/vDVwD2rZBmYHENgZA>`_. Datasets in ``datasets/restricted``
> are restricted and require an explicite request to gain access. Please `submit
> a support ticket <https://milaquebec.freshdesk.com/a/tickets/new>`_ mentionning
> the dataset's access group (ex.: ``scannet_users``) and your cluster's
> username. You can find the dataset's access group by listing the content of
> ``/network/datasets/restricted`` with the `ls command
> <https://cli-cheatsheet.readthedocs.io/en/latest/#ls>`_.

Author: ```suggestion
<https://forms.gle/vDVwD2rZBmYHENgZA>`_.

Datasets in ``datasets/restricted`` are restricted and require an explicite
request to gain access. Please `submit a support ticket
<https://milaquebec.freshdesk.com/a/tickets/new>`_ mentionning the dataset's
access group (ex.: ``scannet_users``), your cluster's username and the
approbation of a member of the group. You can find the dataset's access group
by listing the content of ``/network/datasets/restricted`` with the `ls command
<https://cli-cheatsheet.readthedocs.io/en/latest/#ls>`_.
```
